users/tests.py

```diff
@@ -1 +1,48 @@
-# Create your tests here.
+from rest_framework.test import APIClient
+from rest_framework.test import APITestCase
+from users.models import OneTimePassword, User
+from plio.settings import API_APPLICATION_NAME
+from oauth2_provider.models import Application
+from django.core.management import call_command
+from django.urls import reverse
+from rest_framework import status
+
+
+class BaseTestCase(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+        call_command("pliosuperuser")
+        self.super_user = User.objects.first()
+        Application.objects.create(
+            name=API_APPLICATION_NAME,
+            redirect_uris="",
+            user=self.super_user,
+            client_type=Application.CLIENT_CONFIDENTIAL,
+            authorization_grant_type=Application.GRANT_AUTHORIZATION_CODE,
+        )
+
+
+class UserAuthenticationTestCase(BaseTestCase):
+    def setUp(self):
+        super().setUp()
+        self.user_mobile = "+919876543210"
```

Comment:
shouldn't we make this some env variable? 
Also, if we detach SNS from this using SMS_DRIVER - we won't be able to guarantee that it works right? is this how people usually test an SMS functionality?

Reply:
Yeah, this is just a unit test case that assumes the third-party service works fine. We will have different test cases to test APIs.

It's a random mobile number (9 to 0 in decreasing order). Only for testing. If needed, we can have it as a variable to reuse but I don't think it fits as env variable.